### ISSUE TITLE
argo cannot get resource configmaps in API group "" in the namespace "kubeflow"

### ISSUE THREAD

**Issue by mshyeon** [kubeflow/manifests#1985]
After install, i found the issue with "workflow-controller"
```sh
$ kubectl get pods -n kubeflow

NAME                                                        READY   STATUS             RESTARTS   AGE
...
workflow-controller-58dfc48f94-ggvbp                        1/2     CrashLoopBackOff   72         5h48m
```

- workflow-controller-58dfc48f94-ggvbp  logs
```sh
$ k logs -n kubeflow workflow-controller-58dfc48f94-ggvbp

time="2021-08-23T10:15:10Z" level=info msg="index config" indexWorkflowSemaphoreKeys=true
time="2021-08-23T10:15:10Z" level=info msg="cron config" cronSyncPeriod=10s
time="2021-08-23T10:15:10.423Z" level=info msg="config map" name=workflow-controller-configmap
time="2021-08-23T10:15:10.428Z" level=info msg="Get configmaps 403"
time="2021-08-23T10:15:10.428Z" level=fatal msg="Failed to register watch for controller config map: configmaps \"workflow-controller-configmap\" is forbidden: User \"system:serviceaccount:kubeflow:argo\" cannot get resource \"configmaps\" in API group \"\" in the namespace \"kubeflow\""
```

And I checked serviceaccount(argo), clusterrole(argo-cluster-role), and clusterrolebinding(argo-binding) related in upper.
```sh
$ k describe serviceaccount -n kubeflow argo

$ k describe clusterrole -n kubeflow argo-cluster-role
PolicyRule:
  Resources                                        Non-Resource URLs  Resource Names  Verbs
  ---------                                        -----------------  --------------  -----
  ...
  configmaps                                       []                 []              [get watch list]

$ k describe clusterrolebinding -n kubeflow argo-binding
...
Role:
  Kind:  ClusterRole
  Name:  argo-cluster-role
Subjects:
  Kind            Name  Namespace
  ----            ----  ---------
  ServiceAccount  argo  kubeflow
```
it seems proper settings.

Is there any way to solve this? plz help me..

**Comment by mshyeon** [comment]
I used manifests code with commit `54aa7aea1b3c27d8dbe83dcb49f5877661430639`

**Comment by andreyvelich** [comment]
I had the similar issue deploying the latest version of Kubeflow manifests.
cc @kubeflow/wg-manifests-leads @Bobgy @zijianjoy

**Comment by Bobgy** [comment]
This is a weird issue.
@andreyvelich @mshyeon can you further verify the RBAC permission with:
> Use "kubectl auth can-i" command to verify whether a service account can do a certain thing.
Example usage:
kubectl auth can-i <verb> <resource> --as=system:serviceaccount:<namespace>:<serviceaccountname> [-n <namespace>]
kubectl auth can-i --list --as=system:serviceaccount:<namespace>:<serviceaccountname> [-n <namespace>]

**Comment by Bobgy** [comment]
At the same time, let me try to verify on my end.

**Comment by shashikant-ghangare** [comment]
I too had the same issue and resolved it by using the kubeflow pipeline manifests from https://github.com/kubeflow/pipelines/tree/1.7.0-rc.4/manifests/kustomize . I think the upstream version of kubeflow pipelines has been updated to 1.7.0-rc.4.

**Comment by Bobgy** [comment]
https://github.com/kubeflow/pipelines/commit/7cfd4623f803aff4ec35f48cc5a38806ad62b96d might be the fix between 1.7.0-rc.3 and 1.7.0-rc.4

**Comment by andreyvelich** [comment]
Thank you for investigating this @Bobgy and @shashikant-ghangare.
As I can see in manifests we are using 1.7.0-rc.3 release: https://github.com/kubeflow/manifests/blob/master/apps/pipeline/upstream/base/pipeline/kustomization.yaml#L40.

**Comment by ConverJens** [comment]
@Bobgy Just to clarify, if upgrading to kfp 1.7, some other manifest also needs updating to ensure KubeFlow works correctly?

**Comment by thesuperzapper** [comment]
@Bobgy just checking, has this been resolved?

**Comment by Bobgy** [comment]
Yes, it's fixed in 1.7.0
